src/components/page-types/formPage/tripNotifyMe.js

```diff
@@ -54,31 +52,31 @@ const TripNotifyMe = (props) => {
       <Grid xs={12}>
         <GridCell xs={12} md={6} lg={12}>
           <SbLink
-            classes="su-group su-inline-block su-font-light su-rs-mb-6 su-no-underline su-transition-colors"
+            classes="su-group su-inline-block su-font-semibold su-basefont-23 su-rs-mb-6 su-no-underline su-transition-colors su-text-digital-red-xlight hocus:su-text-white hocus:su-underline hocus:su-underline-offset-[3px]"
             link={{ url: tripURL }}
           >
             <HeroIcon
               iconType="arrow-left"
-              className="su-inline-block su-text-digital-red-light group-hocus:su-text-cardinal-red"
+              className="su-inline-block su-text-digital-red-xlight group-hocus:su-text-white"
               isAnimate
             />
             Back to {tripTitle}
           </SbLink>
           <Helmet titleTemplate={title} title={title} />
           <Heading
             level={1}
+            size={6}
             align="left"
             font="serif"
             id="page-title"
             className="su-drop-shadow"
```

Comment:
```suggestion
            className="su-text-shadow-lg su-rs-mb-2"
```

Using text shadow utility from decanter instead of the drop shadow filter (though color is still a bit light so we might want to darker this a bit further - but you can Bhavika can sort that out 😄 )
Also, tweaking spacer below H1
![Screen Shot 2022-07-08 at 11 48 45 AM](https://user-images.githubusercontent.com/42749717/178054483-6559ed91-be32-4922-9b8e-fcf5fac8a342.png)